assets/global.js

```diff
@@ -408,6 +408,7 @@ class HeaderDrawer extends MenuDrawer {
     this.header = this.header || document.getElementById('shopify-section-header');
     this.borderOffset = this.borderOffset || this.closest('.header-wrapper').classList.contains('header-wrapper--border-bottom') ? 1 : 0;
     document.documentElement.style.setProperty('--header-bottom-position', `${parseInt(this.header.getBoundingClientRect().bottom - this.borderOffset)}px`);
+    document.querySelector('#shopify-section-header').classList.add('menu-open');
```

Comment:
You can replace this with `this.header` from line 408
```suggestion
    this.header.classList.add('menu-open');
```